easybuild/easyconfigs/m/MPB/MPB-1.11.1-foss-2020b.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'ConfigureMake'
+
+name = 'MPB'
+version = '1.11.1'
+
+homepage = 'https://mpb.readthedocs.io/en/latest/'
+description = """MPB is a free and open-source software package for computing
+ the band structures, or dispersion relations, and electromagnetic
+ modes of periodic dielectric structures, on both serial
+ and parallel computers. MPB is an acronym for MIT Photonic Bands."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['https://github.com/NanoComp/%(namelower)s/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['dc55b081c56079727dac92d309f8e4ea84ca6eea9122ec24b7955f8c258608e1']
+
+builddependencies = [
+    ('Autotools', '20200321'),
+]
+dependencies = [
+    ('Python', '3.8.6'),
+    ('HDF5', '1.10.7'),
+    ('libctl', '4.1.3'),
+    ('Guile', '2.2.4'),
+    ('libreadline', '8.0'),
+]
+
+local_configopts = '--with-pic --with-blas=openblas --with-lapack=openblas --with-libctl=$EBROOTLIBCTL/share/libctl'
+local_configopts = local_configopts + "--enable-shared"
+configopts = [
+    local_configopts,
+    local_configopts + "--with-inv-symmetry",
+    local_configopts + "--with-mpi",
+    local_configopts + "--with-mpi --with-inv-symmetry",
+]
+
+local_sanity_checks = "'bin/%(namelower)s', 'bin/%(namelower)s-data', 'bin/mpbi', 'bin/mpbi-data', 'bin/mpbi-mpi', "
+local_sanity_checks = local_sanity_checks + "'bin/mpbi-split', 'bin/%(namelower)s-mpi', "
+local_sanity_checks = local_sanity_checks + "'lib/libmpbi_mpi.a', 'lib/libmpbi.a', 'lib/libmpb_mpi.a', "
+local_sanity_checks = local_sanity_checks + "'lib/libmpb.so', 'lib/libmpbi_mpi.so', 'lib/libmpbi.so', "
+local_sanity_checks = local_sanity_checks + "'bin/%(namelower)s-split', 'lib/libmpb.a', "
+local_sanity_checks = local_sanity_checks + "'lib/libmpb_mpi.so'"
```

Comment:
This is likely a better way do handle this.
local_sanity_checks = ['bin/%s' % x for x in ['%(namelower)s', '%(namelower)s-data', 'mpbi', 'mpbi-data'...]]
l_s_c += ['lib/lib%s.%s' % (x,y) for y in ['a', SHLIB_EXT] for x in ['mpbi_mpi', ...]]

sanity_check_paths = {
  'files': local_sanity_checks,
  'dirs': ['include'],
}